### DISCUSSION TITLE
Apex Donut Chart with rounded Numbers?

### DISCUSSION THREAD

**Discussion by AgentHawk** [apexcharts/apexcharts.js#3302] [Q&A] [answered]
Hey im trying to round my Output....

![apex_home](https://user-images.githubusercontent.com/25323080/183364086-aad73a82-356c-4562-bb56-1ef0f39ec273.jpg)

`
type: custom:apexcharts-card
chart_type: donut
apex_config:
  plotOptions:
    pie:
      donut:
        background: transparent
        labels:
          show: true
          total:
            show: lafeslaftrue
header:
  show: true
  title: Hausverbrauch
  show_states: true
  colorize_states: true
series:
  - entity: sensor.openwb_daily_yield_house
    name: Remaining
    transform: >-
      return parseFloat(x) -
      parseFloat(hass.states['sensor.stecker5_energy_today'].state) -
      parseFloat(hass.states['sensor.usb2_energy_today'].state) -
      parseFloat(hass.states['sensor.usb3_energy_today'].state) -
      parseFloat(hass.states['sensor.stecker7_energy_today'].state) -
      parseFloat(hass.states['sensor.tasmota_server_energy_today'].state) -
      parseFloat(hass.states['sensor.stecker1_energy_today'].state)
  - entity: sensor.stecker5_energy_today
    name: Waschmaschiene
  - entity: sensor.usb2_energy_today
    name: Spühlmaschiene
  - entity: sensor.usb3_energy_today
    name: TV Manu
  - entity: sensor.stecker7_energy_today
    name: TV Wohnzimmer
  - entity: sensor.tasmota_server_energy_today
    name: Server
  - entity: sensor.stecker1_energy_today
    name: Trockner
`

**Comment by AgentHawk** [comment]
no one?

**Comment by lebequin** [comment, marked as answer]
you can add a formatter below your label property:
`formatter: function () {
      let sum = 0;
      for (let i = 0; i < array.length; i++) {
        sum += array[i];
      }
      let average = sum / array.length;
      return (average.toFixed(2) + '%');
}`
